tests/ogc_tests.rs

```diff
@@ -4,22 +4,16 @@ use std::fs;
 use std::path::PathBuf;
 
 pub fn validate_file(f: &str) {
-    //println!("Current Directory: {:#?}", env::current_dir());
     println!("File Path: {:#?}", f);
     let cql2 = fs::read_to_string(f).unwrap();
     println!("CQL2: {}", cql2);
     let expr: cql2_rs::Expr = parse(&cql2);
-    println!("Expr: {}", expr.as_json_pretty());
-    let valid = expr.validate();
-    assert!(valid)
-}
+    println!("Expr: {}", expr.to_json_pretty().unwrap());
 
-#[rstest]
-fn json_examples_are_valid(#[files("tests/fixtures/json/*.json")] path: PathBuf) {
```

Comment:
I removed the `json_examples_are_valid` rstest function because AFAICT these cases are covered by `for_each_json_file`.